src/Basque/Mews.Fiscalizations.Basque.Tests/Bizkaia/LROERequestDtoValidationTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Mews.Fiscalizations.Core.Xml;
+using System.Text;
+
+namespace Mews.Fiscalizations.Basque.Tests.Bizkaia
+{
+    [TestFixture]
+    public class LROERequestDtoValidationTests
+    {
+        private const string BatuzXsdFilename = "./Xsd/LROE_PJ_240_1_1_FacturasEmitidas_ConSG_AltaPeticion_V1_0_2.xsd";
+        private const string BatuzTiposComplejosFilename = "./Xsd/batuz_TiposComplejos.xsd";
+        private const string BatuzEnumeradosFilename = "./Xsd/batuz_Enumerados.xsd";
+        private const string BatuzTiposBasicosFilename = "./Xsd/batuz_TiposBasicos.xsd";
+
+        [Test]
+        public void CreateTBatuzInvoiceDto_XmlSerialization_Succeeds()
+        {
+            var batuzInvoiceRequest = LROERequestHelper.CreateSampleBatuzRequest();
+
+            Assert.DoesNotThrow(() =>
+            {
+                var xmlElement = XmlSerializer.Serialize(batuzInvoiceRequest, new XmlSerializationParameters(
+                    encoding: Encoding.UTF8,
+                    namespaces: Array.Empty<XmlNamespace>()));
```

Comment:
The serialization params are optional
```suggestion
                var xmlElement = XmlSerializer.Serialize(batuzInvoiceRequest);
```